inc/wpseo-non-ajax-functions.php

```diff
@@ -236,7 +236,7 @@ function wpseo_admin_bar_menu() {
 
 	$focuskw = '';
 	$score   = '';
-	$seo_url = get_admin_url( null, 'admin.php?page=' . Yoast_Alerts::ADMIN_PAGE );
+	$seo_url = get_admin_url( null, 'admin.php?page=wpseo_dashboard' );
```

Comment:
Please introduce a different constant to save this value in. That constant can then have a comment above it with "Never change this because it breaks compatibility with all add-ons."